.github/workflows/ci_cpu.yml

```diff
@@ -21,9 +21,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macOS-latest]
+        os: [ubuntu-20.04, macOS-latest, macos-14]
     steps:
-      - name: Setup Python 3.8
+      - name: Setup Python for M1
```

Comment:
Why python 3.10? It'll make releasing the binaries a bit annoying if each platform requires a different python version

Reply:
Github `macos-14` does not work with python setup less than 3.10 https://github.com/actions/setup-python/issues/808

Reply:
Reading through that issue it seems like the issue was fixed, mind retriggering with 3.8